src/Components/Calculator.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+class Calculator extends React.PureComponent {
+  render() {
+    return (
```

Comment:
**Code Review Feedback: Calculator Component**

- [ ] Great job on your implementation of the `Calculator` component! However, I noticed that the component currently doesn't have any child components. To fulfill the requirement of having at least one child component, you can consider breaking down the different sections of your calculator into separate reusable components.

Here's an example of how you can refactor your code to include child components:

```jsx
import React from 'react';
import Keypad from './Keypad';

class Calculator extends React.PureComponent {
  render() {
    return (
      <section className="cont">
        <div className="container">
          <form>
            <input type="text" placeholder="0" />
          </form>

          <Keypad />
        </div>
      </section>
    );
  }
}

export default Calculator;
```

In this example, I've created a new component called `Keypad`, which represents the section containing all the buttons. By extracting this part into its own component, you achieve better code organization and improve reusability.

Feel free to create more child components if needed, based on the different logical sections of your calculator.

Keep up the good work, and if you have any further questions, feel free to ask!